Rovers/Rovers.tex

```diff
@@ -0,0 +1,387 @@
+Each improvement would be on its own timeline with milestones.
+
+\begin{table}[ht!]
+    \caption{Timeline}
+    \centering
+    {\renewcommand{\arraystretch}{1.5}
+    \begin{tabularx}{\linewidth}{LL} 
+    \hline
+    \textbf{Time} & \textbf{Phase} \\
+    \hline
+    August & Learning and Planning Phase \\
+    September - November & Design Phase  \\
+    December - February & Manufacturing Phase \\
+    March - May & Testing Phase \\
+    \hline
+    \end{tabularx}
+    }
+\label{tab:timeline}
+\end{table}
+
+\subsubsection{Learning and Planning Phase}
+\label{subsubsec:phaseone}
+Learn about rovers and create a set of plans to help guide the team through the execution and closure phases of the project. Requirements that are associated with a project result are specified as clearly as possible.
+\subsubsection{Design Phase}
+\label{subsubsec:phasetwo}
+Designs are developed, with which the project result can apparently be achieved. Depending on the subject of the project, the products of the design phase can include dioramas, sketches, flow charts, site trees, HTML screen designs, prototypes, photo impressions, UML schemas, and CAD models created.
+\subsubsection{Manufacturing and Construction Phase}
+\label{subsubsec:phasethree}
+Construction of the actual project result. Programmers are occupied with encoding, engineers are machining and building, the actual reorganization takes place.
+\subsubsection{Testing Phase}
+\label{subsubsec:phasefour}
+Formally controlled and focused testing is performed to uncover errors and bugs in the IT solution that need to be resolved. This includes software unit testing and a mock URC-like competition. 
+
+\section{Externalities}
+\label{sec:externalities}
+\subsection{Prerequisite Skills}
+\label{subsec:skills}
+The project is going to require members that are knowledgeable in mechanical design and manufacturing, software engineering, (potentially) machine learning, electrical engineering, and project management. 
+These members will also need to be experienced in research and design document work. 
+Though this seems simple, finding SPEX members to do this well may be a challenge due to competition from other projects. 
+Building a rover, especially one built to URC specifications requires a large team with a diverse set of technical skills. 
+This project will require SPEX to look at and asses what areas are sufficient and what areas need to be developed in order to build a rover. 
+It will also let us know to what spec we are currently capable of building such a rover. 
+
+\subsection{Funding Requirements}
+\label{subsec:funding}
+The rover project is looking at a minimal funding amount of \$500.00 USD from RIT SPEX. This funding is needed to make a rover that fulfills the MVP defined in the Primary Objective section. In order to give a better deliverable the team will also be reaching out to a large list of companies looking for further funding through sponsorship. 
```

Comment:
Similar to a comment above, the baseline for this cost estimate was given during the PDD review but is not in the paper itself. I think it is important to include that here.